ramls/pieces.raml

```diff
@@ -47,6 +60,42 @@ resourceTypes:
         example: true
         required: false
         default: false
+  /batch:
+    delete:
+      description: Deletes multiple pieces optionally including related holdings.
+      body:
+        application/json:
+          type: string
+          example: |
+            {
+              "ids": ["123", "456", "789"],
+              "deleteHoldings": false
+            }
+      responses:
+        204:
+          description: "Pieces records successfully deleted"
```

Comment:
`Piece records` or `Pieces ` more correct

Reply:
Also refactor Sentences in other places to be gramatically correct

Reply:
thank you, sure will.